packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModalContainer.js

```diff
@@ -0,0 +1,106 @@
+import { inject, observer } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import CollectionsModal from './CollectionsModal'
+import SelectCollection from './components/SelectCollection'
+import CreateCollection from './components/CreateCollection'
+
+function storeMapper (stores) {
+  const { addSubjects, collections, createCollection, searchCollections } = stores.store.collections
+  return {
+    addSubjects,
+    collections,
+    createCollection,
+    searchCollections
+  }
+}
+
+@inject(storeMapper)
+@observer
+class CollectionsModalContainer extends Component {
+  constructor () {
+    super()
+    this.addToCollection = this.addToCollection.bind(this)
+    this.close = this.close.bind(this)
+    this.createCollection = this.createCollection.bind(this)
+    this.onSelect = this.onSelect.bind(this)
+    this.updateCollection = this.updateCollection.bind(this)
+    this.state = {
+      active: false,
+      newCollection: {
+        display_name: '',
+        private: false
+      },
+      selectedCollectionId: null
+    }
+  }
+
+  componentDidMount () {
+    this.setState({ active: true })
```

Comment:
This is only here for testing. At some point, the Classifier and Recent Subjects components will have to be connected up to toggle this.